src/platforms/java/migration.mdx

```diff
@@ -4,6 +4,12 @@ sidebar_order: 1000
 description: "Learn about migrating from io.sentry:sentry 1.x to io.sentry:sentry 3.x."
 ---
 
+## Migrating from `io.sentry:sentry` `4.3.0` to `io.sentry:sentry` `5.0.0`
+
+### Spring Boot
+
+- the property `sentry.enable-tracing` is deprecated. To enable tracing simply set `sentry.traces-sample-rate` or create a bean implementing `TracesSamplerCallback`
```

Comment:
this could also be a different PR, unless `Enable Tracing` is already deprecated on 4.3.0 so it'd be fine to merge as it is

Reply:
I am not sure what do you mean

Reply:
what I mean is, the migration page is necessary to be live in the docs, so people can try the betas

but the removal from platforms/common/performance/index.mdx  cannot be merged yet, because its not stable yet, so they should be different PRs.

Reply:
Ok, done: https://github.com/getsentry/sentry-docs/pull/3522